fastlane/Fastfile

```diff
@@ -275,63 +162,50 @@ platform :ios do
     )
 
     match(
-      type: "development",
+      type: options[:match_type],
       readonly: true,
       keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
       keychain_password: ENV["MATCH_KEYCHAIN_PASSWORD"],
     )
 
-    # Build for appetize
-    build_app(
-      scheme: "PrimerSDK-Example",
-      sdk: "iphonesimulator", # Appetize needs a simulator app
-      workspace: "./Example/PrimerSDK.xcworkspace",
-      configuration: "Debug",
-      export_method: "development",
-      archive_path: "/var/tmp/PrimerSDK_Example.xcarchive",
-      # Build speed optimisation
-      skip_package_ipa: true,
-      skip_package_pkg: true,
-    )
+  end
 
-    # Appetize needs the .app to be zipped
-    zip(
-      path: "/var/tmp/PrimerSDK_Example.xcarchive/Products/Applications/PrimerSDK_Example.app",
-      output_path: "./PrimerSDK_Example.zip",
-      symlinks: true
-    )
+  desc 'This action sets the version and build number'
+  private_lane :set_version_and_build_number do
 
-    # Find public key of appetize
-    uri = URI('https://livedemostore.primer.io/appetize/ios/' + version_name)
-    public_key = Net::HTTP.get(uri)
-    puts "public_key: " + public_key
+    # We don't really need the version number
+    # at this moment.
+    # The Build number is the unique identifier of the package
+    # matching the Github Workflow run ID and number
+    
+    # Set version number
+    # increment_version_number(
+    #   version_number: sdk_version_name_source_branch,
+    #   xcodeproj: app_xcode_proj
+    # )
 
-    # Upload to Appetize as it's version number
-    appetize(
-      path: "./PrimerSDK_Example.zip",
-      platform: "ios",
-      api_token: appetize_api_token,
-      public_key: public_key,
-      note: version_name
+    # Set build number
+    increment_build_number(
+      build_number: "#{github_run_id}.#{github_run_number}",
+      xcodeproj: app_xcode_proj
     )
 
-    # Build for firebase
```

Comment:
Is Firebase upload completely removed?

Reply:
It's still there but comment out as we will need to work the `FIREBASE_API_TOKEN` which is now required and update this function

```
    # Utilize the firebase client token to store in CI
    # Upload to Firebase
    # firebase_app_distribution(
    #   service_credentials_file: "firebase_credentials.json",
    #   app: "1:1024117832450:ios:d8c2e74c66341d8cf3201b",
    #   groups: "primer-internal",
    #   release_notes: "QA release: " + sdk_version_name_source_branch + " [" + str_time + "]"
    # )

```